lib/user-interface/react-app/src/common/types.ts

```diff
@@ -137,8 +138,26 @@ export interface DocumentItem {
   subDocuments: number;
   createdAt: string;
   updatedAt: string;
+  rssFeedId?: string;
+  rssLastCheckedAt?: string;
+  crawlerProperties?: {
+    followLinks: boolean;
+    limit: number;
+  }
 }
```

Comment:
can you explain why this information, which is related to the source of the document, must be included in the `DocumentItem` type?

Reply:
On the `rss-feed.tsx` page, both the RSS Feed subscription and the RSS Feed posts display this information.
For the Feed, it's showing the currently configured crawler properties that will be leveraged by website crawling requests triggered by detecting new feed posts. 
On the individual feed records, it shows what was configured when the feed post was ingested and sent for crawling. 
![sample](https://github.com/aws-samples/aws-genai-llm-chatbot/assets/40297962/758ecfbe-f902-454a-bf80-2039f79c9d90)